### DISCUSSION TITLE
[zi-vim-syntax] Commented out lines are still highlighted

### DISCUSSION THREAD

**Discussion by Farzat07** [z-shell/zi#237] [General]
This is the first plugin I saw doing this. Makes me feel weird, kinda as if the line is still gonna do something.

Afaik, highlighted comments are supposed to be lines which have a function even if they appear to be commets, such as shebangs or #compdef at the beginning of a completion file.

**Comment by ss-o** [comment]
Hi @Farzat07, I will look at this. Thanks for sharing :dart:
